master-firmware/src/robot_parameters.h

```diff
@@ -11,6 +11,7 @@ extern "C" {
  * Robot base parameters
  */
 #define ROBOT_SIZE_X_MM                     205 // Front to back length
+#define DEFAULT_OPPONENT_SIZE_MM            300
```

Comment:
Yes, we need to unify all parameters and choose how it will support the robot specific parameters.